test/notify.js

```diff
@@ -21,7 +21,7 @@ function Control(shouldNotifyInNpmScript) {
 const setupTest = isNpmReturnValue => {
 	['..', 'is-npm'].forEach(clearModule);
 	process.stdout.isTTY = true;
-	mock('is-npm', isNpmReturnValue || false);
+	mock('is-npm', {isNpm: isNpmReturnValue || false});
```

Comment:
The API of [is-npm@3.0.0](https://github.com/sindresorhus/is-npm) has been changed. So I updated the mock export.